### ISSUE TITLE
[TablePagination] Row per page showing label missing in wave accessibility tool

### ISSUE THREAD

**Issue by Shirishas11** [mbrn/material-table#2848]
Hi I am using Material-Table component with all possible option as given in the code however when it comes to accessibility tool wave on 'Row per page' it says label is missing from the accessibility point of view. How do i fix this.

[Screenshot](https://imgur.com/SAcABG4)

```
import React, { memo, useState, useMemo } from "react";
import { compose } from "redux";
import { withStyles } from "@material-ui/core/styles";

import MaterialTable from "material-table";

const styles = (theme) => ({
  materialTableContainer: {
    "& .MuiButtonBase-root.Mui-disabled": {
      wordBreak: "keep-all",
      "& > svg": {
        display: "none"
      }
    },
    "& .MuiTableCell-head": {
      zIndex: 0
    }
  }
});

export function rotterDamTable(props) {
  const {
    classes,
    params,
    dataArray,
    dataTotal,
    page,
    rowsPerPage,
    title,
    columns,
    actions,
    isStatic,
    tableOptions,
    ...rest
  } = props;

  const [pageState, setPage] = useState(page || 0);
  const [rowsPerPageState, setRowsPerPage] = useState(rowsPerPage || 10);


  const filtered_columns = useMemo(() => {
    if (props.handleSearching)
      return columns.map((column) => ({
        customFilterAndSearch: () => true,
        ...column
      }));
    return columns;
  }, [columns, props.handleSearching]);

  const currentPage = () => (params ? params.page : pageState);
  const currentRowsPerPage = () => (params ? params.count : rowsPerPageState);

  const handleChangePage = (newPage) => {
    setPage(newPage);
    props.handlePagination({ count: currentRowsPerPage(), page: newPage });
  };

  const handleChangeRowsPerPage = (pageSize) => {
    setRowsPerPage(+pageSize);
    props.handlePagination({ count: +pageSize, page: currentPage() });
  };

  const handleOrderChange = (columnId, orderDir) => {
    props.handleOrdering({
      orderBy: columnId === -1 ? "id" : columns[columnId].field,
      orderDir: columnId === -1 ? "asc" : orderDir
    });
  };

  const handleSearchChange = (searchText) =>
    props.handleSearching && props.handleSearching(searchText);

  return (
    <div className={classes.materialTableContainer}>
      <MaterialTable
        data={dataArray || []}
        totalCount={dataTotal || 0}
        page={currentPage()}
        title="jsdljfldjf"
        onChangePage={handleChangePage}
        onChangeRowsPerPage={handleChangeRowsPerPage}
        onOrderChange={handleOrderChange}
        onSearchChange={handleSearchChange}
        onRowClick={props.handleRowClick}
        columns={filtered_columns}
        actions={actions}
        options={{
          search: !!props.handleSearching, // Disable searching when callback isn't passed
          actionsColumnIndex: -1,
          pageSize: currentRowsPerPage(),
          emptyRowsWhenPaging: false,
          debounceInterval: 1000,
          ...tableOptions
        }}
        localization={{
          header: {
            actions: "actions"
          },
          pagination: {
            labelRowsSelect: "labelRowsSelect",
            labelDisplayedRows: "labelDisplayedRows",
            labelRowsPerPage: "labelRowsPerPage",
            firstAriaLabel: "firstAriaLabel",
            firstTooltip: "firstTooltip",
            previousAriaLabel: "previousAriaLabel",
            previousTooltip: "previousTooltip",
            nextAriaLabel: "nextAriaLabel",
            nextTooltip: "nextTooltip",
            lastAriaLabel: "lastAriaLabel",
            lastTooltip: "lastTooltip"
          }
        }}
        {...rest}
      />
    </div>
  );
}

rotterDamTable.defaultProps = {
  extraTitleProps: {}
};

export default compose(withStyles(styles), memo)(rotterDamTable);
```
